tests/platform_tests/cli/test_show_chassis_module.py

```diff
@@ -33,13 +41,32 @@ def parse_chassis_module(output, expected_headers):
     return result
 
 
-def test_show_chassis_module_status(duthosts, enum_rand_one_per_hwsku_hostname):
+def test_show_chassis_module_status(duthosts, enum_rand_one_per_hwsku_hostname, dut_vars):
     cmd = " ".join([CMD_SHOW_CHASSIS_MODULE, "status"])
     logger.info("verifying output of cli command {}".format(cmd))
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
     exp_headers = ["Name", "Description", "Physical-Slot", "Oper-Status", "Admin-Status"]
     skip_mod_list = get_skip_mod_list(duthost)
     skip_logical_lc_list = get_skip_logical_module_list(duthost)
+    """
+    Gather expected module slot data from a inventory file if 'module_slot_info' is defined in the inventory
+    # Sample inventory with module_slot_info:
+    str-sonic-chassis-01-sup:
+        ansible_host: 10.251.0.188
+        model: SOME-VENDOR-MODEL
+        serial: BADC0FFEE123
+        base_mac: 38:8a:29:13:45:67
+        module_slot_info:
```

Comment:
is this a new variable which you are proposing in inventory file @gechiang ? if so we need to add this into the invetory for this case to be validated  ( in internal repo )

Reply:
Yes @judyjoseph I have raised separate internal PR for our nightly lab chassis.  Have shared that PR with you directly.